pkgs/development/libraries/mlt/qt-5.nix

```diff
@@ -55,7 +58,11 @@ mkDerivation rec {
     ladspa-sdk
     ladspaPlugins
     rubberband
-  ];
+  ] ++ lib.optionals enableGlaxnimate ([
+    # Enable glaxnimate https://github.com/NixOS/nixpkgs/issues/209923;
+    glaxnimate
+    # Not sure why, but mlt needs to compile itself glaxinate, instead of picking the parent project (that, admitingly, only contains the binary)
```

Comment:
These two lines are somewhat contradictory: Either you rebuild glaxnimate here, or you use the build input.